src/main/java/edu/cornell/kfs/module/purap/batch/dataaccess/impl/JaggaerUploadDaoJdbc.java

```diff
@@ -16,20 +18,27 @@
 import edu.cornell.kfs.module.purap.batch.dataaccess.JaggaerUploadDao;
 import edu.cornell.kfs.module.purap.businessobject.lookup.JaggaerContractAddressUploadDto;
 import edu.cornell.kfs.module.purap.businessobject.lookup.JaggaerContractPartyUploadDto;
+import edu.cornell.kfs.sys.CUKFSConstants;
+import edu.cornell.kfs.sys.service.ISOFIPSConversionService;
 import edu.cornell.kfs.vnd.CUVendorConstants.FIELD_NAMES;
 
 import org.springframework.jdbc.core.RowMapper;
 
 public class JaggaerUploadDaoJdbc extends PlatformAwareDaoBaseJdbc implements JaggaerUploadDao {
     private static final Logger LOG = LogManager.getLogger();
-
+    
+    protected ISOFIPSConversionService isoFipsConversionService;
+    
     @Override
     public List<JaggaerContractPartyUploadDto> findJaggaerContractParty(JaggaerContractUploadProcessingMode processingMode, String processingDate) {
         try {
             RowMapper<JaggaerContractPartyUploadDto> rowMapper = (resultSet, rowNumber) -> {
+                if (rowNumber == 0 || rowNumber % 100 == 0) {
```

Comment:
You could simplify this condition to just use the MOD (%) part of the check, since 0 MOD 100 should be zero. The same goes for the similar areas in this file.

Reply:
OK